packages/mobile-app/metro.config.js

```diff
@@ -56,41 +40,63 @@ const emptyModules = new Set([
   "sqlite3",
 ]);
 
-config.resolver.resolveRequest = (context, moduleName, platform) => {
-  if (shims[moduleName]) {
-    return shims[moduleName];
-  }
+module.exports = (() => {
```

Comment:
@dguenther @jowparks  Turning this into an IIFE isn't necessary, but when I was trying to figure out how to set up the appropriate transformers and resolvers to process SVGs, all the examples were using this pattern so figured we might as well.

I'd be totally fine with breaking this out of the IIFE.

I do think that modifying the outermost resolver and transformer objects once is easier vs modifying individual properties (e.g. `config.resolver.resolveRequest = () => {...}` so I would like to keep that if that is cool with y'all.

Reply:
I don't have a strong opinion here as long as everything works. I agree that modifying the resolver/transformer objects once is an improvement too 👍